pyscriptjs/src/pyexec.ts

```diff
@@ -70,17 +70,26 @@ export function displayPyException(err: Error, errElem: HTMLElement) {
     //addClasses(errElem, ['py-error'])
     const pre = document.createElement('pre');
     pre.className = 'py-error';
-
-    if (err.name === 'PythonError') {
-        // err.message contains the python-level traceback (i.e. a string
-        // starting with: "Traceback (most recent call last) ..."
-        logger.error('Python exception:\n' + err.message);
-        pre.innerText = err.message;
-    } else {
-        // this is very likely a normal JS exception. The best we can do is to
-        // display it as is.
-        logger.error('Non-python exception:\n' + err.toString());
-        pre.innerText = err.toString();
+    let errorCount = 0;
+    const errorMsg = err.name === 'PythonError' ? err.message : err.toString();
+    const pyErrorElements = document.getElementsByClassName('py-error');
+    for (const pyErrorElement of pyErrorElements) {
+        if ((pyErrorElement as HTMLElement).innerText.includes(errorMsg)) {
+            errorCount++;
+        }
+    }
+    if (errorCount === 0) {
```

Comment:
If I udnerstand this correctly, this means that if we already got one python exception and errorCount is >1 and we get a JS exception, we will never show the js exception? 🤔

Reply:
Hi @FabioRosado,
The function is actually checking any banner with the current exception message was already displayed on the UI or not (in which case `errorCount` will not be zero), if not then create a banner with the exception message.
So new banners are not created with the same exception message :sweat_smile: